### PR TITLE
Update cascade plots

### PR DESCRIPTION
**Pull request by komalsrathi**
I have made the following changes:

1) Cascade Plots (All genes)

- removed underscore in annotations and legend
- removed `CNS region` from annotation and legend
- Update Cancer Group with `PXA` where `Molecular Subtype == "PXA"`
- replace NA values with "Flag" in Molecular Subtype and Cancer Group
- Update Cancer Group names to the following: 
DHG -> (DHG) Diffuse Hemispheric Glioma
DMG -> (DMG) Diffuse Midline Glioma
HGG -> (HGG) High Grade Glioma (not otherwise specified)
IHG -> (IHG) Infantile High Grade Glioma
PXA -> (PXA) Pleomorphic Xanthoastrocytoma

2) Cascade Plots (Top 20 genes)

- removed underscore in annotations and legend
- removed `CNS region` and `Molecular Subtype` from annotation and legend
- Update Cancer Group with `PXA` where `Molecular Subtype == "PXA"`
- replace NA values with "Flag" in Molecular Subtype and Cancer Group
- Update Cancer Group names to the following: 
DHG -> (DHG) Diffuse Hemispheric Glioma
DMG -> (DMG) Diffuse Midline Glioma
HGG -> (HGG) High Grade Glioma (not otherwise specified)
IHG -> (IHG) Infantile High Grade Glioma
PXA -> (PXA) Pleomorphic Xanthoastrocytoma

Fixes: #89 #90

### CODE REVIEW

**Comment by mkoptyra** [comment]
@jharenza would you be able to help with this review too?

File: analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_allgenes.R [hunk 1]
```diff
@@ -42,68 +42,88 @@ alter_fun = list(
...
+# Cancer_Group should be PXA where Molecular_Subtype is PXA
+annot_info <- annot_info %>%
+  mutate(Cancer_Group = ifelse(Molecular_Subtype == "PXA", "PXA", Cancer_Group))
+
+# long names for Cancer_Group
+annot_info <- annot_info %>%
+  mutate(Cancer_Group = case_when(Cancer_Group == "DHG" ~ "(DHG) Diffuse Hemispheric Glioma",
+                                  Cancer_Group == "DMG" ~ "(DMG) Diffuse Midline Glioma",
+                                  Cancer_Group == "HGG" ~ "(HGG) High Grade Glioma (not otherwise specified)",
+                                  Cancer_Group == "IHG" ~ "(IHG) Infantile High Grade Glioma",
```
Comment (jharenza) [inline analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_allgenes.R:76]:
```suggestion
                                  Cancer_Group == "IHG" ~ "(IHG) Infantile Hemispheric Glioma",
```

File: analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_allgenes.R [hunk 2]
```diff
@@ -42,68 +42,88 @@ alter_fun = list(
...
+                       "Cerebellar" = "#94004C"),
+  `CNS region` = c("Posterior fossa" = "#D4806C",
+                   "Other" = "#7C8F97",
+                   "Midline" = "#344C68",
+                   "Hemispheric" = "#94004C",
+                   "Mixed" = "darkgreen"),
+  `Cancer Group` = c("(DMG) Diffuse Midline Glioma" = "#053061",
+                     "(DHG) Diffuse Hemispheric Glioma" = "#A6761D",
+                     "(HGG) High Grade Glioma (not otherwise specified)" = "#4393c3",
+                     "(IHG) Infantile High Grade Glioma" = "#E7298A",
```
Comment (jharenza) [inline analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_allgenes.R:124]:
```suggestion
                     "(IHG) Infantile Hemispheric Glioma" = "#E7298A",
```

File: analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_top20genes.R [hunk 3]
```diff
@@ -42,73 +42,94 @@ alter_fun = list(
...
+
+# remove Molecular_Subtype
+annot_info$Molecular_Subtype <- NULL
+
+# long names for Cancer_Group
+annot_info <- annot_info %>%
+  mutate(Cancer_Group = case_when(Cancer_Group == "DHG" ~ "(DHG) Diffuse Hemispheric Glioma",
+                                  Cancer_Group == "DMG" ~ "(DMG) Diffuse Midline Glioma",
+                                  Cancer_Group == "HGG" ~ "(HGG) High Grade Glioma (not otherwise specified)",
+                                  Cancer_Group == "IHG" ~ "(IHG) Infantile High Grade Glioma",
```
Comment (jharenza) [inline analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_top20genes.R:84]:
```suggestion
                                  Cancer_Group == "IHG" ~ "(IHG) Infantile Hemispheric Glioma",
```

File: analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_top20genes.R [hunk 4]
```diff
@@ -42,73 +42,94 @@ alter_fun = list(
...
+                       "Cerebellar" = "#94004C"),
+  `CNS region` = c("Posterior fossa" = "#D4806C",
+                   "Other" = "#7C8F97",
+                   "Midline" = "#344C68",
+                   "Hemispheric" = "#94004C",
+                   "Mixed" = "darkgreen"),
+  `Cancer Group` = c("(DMG) Diffuse Midline Glioma" = "#053061",
+                     "(DHG) Diffuse Hemispheric Glioma" = "#A6761D",
+                     "(HGG) High Grade Glioma (not otherwise specified)" = "#4393c3",
+                     "(IHG) Infantile High Grade Glioma" = "#E7298A",
```
Comment (jharenza) [inline analyses/oncoplots/07-cascade_plots_add_tumor_only_two_age_groups_top20genes.R:130]:
```suggestion
                     "(IHG) Infantile Hemispheric Glioma" = "#E7298A",
```

**Review by jharenza — APPROVED** [review]
One minor update request - IHG == Infantile Hemispheric Glioma.

For TMB, I noticed the scale is 0-20. Assuming units are Mut/Mb. Might want to add units. Is it correct that 20 is max? Are any samples ultra-hypermutant, or not in this cohort? An alternative (and can be addressed in another PR) would be to assign the samples as TMB == Normal (<10 Mut/Mb), Hypermutant >=10 but <100, Ultra-hypermutant >=100).

Approving since I am OOO, so long as IHG is fixed here.

cc @mkoptyra

**Comment by komalsrathi** [comment]
> For TMB, I noticed the scale is 0-20. Assuming units are Mut/Mb. Might want to add units. Is it correct that 20 is max? Are any samples ultra-hypermutant, or not in this cohort? An alternative (and can be addressed in another PR) would be to assign the samples as TMB == Normal (<10 Mut/Mb), Hypermutant >=10 but <100, Ultra-hypermutant >=100).

yes 20.65 is max. Yes, units are `Mut/Mb`.

**Comment by komalsrathi** [comment]
Can discuss your comment in the upcoming meeting.

**Pull request merged**